app/Http/HttpHelper.php

```diff
@@ -70,7 +70,7 @@ public static function serializeEmptyObjectsControllerLevel(array $data): array
 
     public static function getRouteName(Request $request) : string
     {
-        $route = explode('\\', $request->route()[1]['uses']);
+        $route = explode('\\', $request->route()[1]['uses'] ?? 'App\Undefined');
```

Comment:
Afaik route is empty for a reason, and it should throw an exception and we should investigate why it happens. This behaves differently depending whether you run the app through artisan directly or as a PSR worker. I've fixed a bug recently which caused this bit to throw an exception.

Reply:
@pushrbx Should be fine to return Undefined in this instance though?

Reply:
I think I've missed this comment. The goal here is to get an exception at runtime with the entirely empty value, or at least it should throw an error if `$request->route()[1]['uses']` is null, because that's a fatal error, it should never be empty. If it's empty then we've made a programming error / programmer error somewhere, so we need a canary against it. 😄